dotCMS/src/main/java/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -782,7 +783,6 @@ public boolean copyFolder ( String inode, String newFolder ) throws Exception {
      * @return Confirmation message
      * @throws Exception
      */
-    @WrapInTransaction
```

Comment:
why delete this annotation?

Reply:
None of the methods in this Ajax class have this annotation and this was actually causing an issue of not letting the API handle the rolling back in case of an exception at API level, thus leaving the system in a bad state.

Reply:
it is calling just one API, it would be ok